test/boulder-tools/build.sh

```diff
@@ -40,6 +44,16 @@ GOBIN=/usr/local/bin GOPATH=/tmp/gopath go get \
 
 wait
 
+# Install pkcs11-proxy
+git clone https://github.com/SUNET/pkcs11-proxy && \
+  cd pkcs11-proxy && \
+  git checkout 944684f78bca0c8da6cabe3fa273fed3db44a890 && \
```

Comment:
Oops, you're right - I forgot that this only works with branches, not arbitrary refs.

BTW @rolandshoemaker, why'd you pick this particular commit? Can you add a comment?